js/actions/windowActions.js

```diff
@@ -738,6 +738,17 @@ const windowActions = {
   },
 
   /**
+   * Sets the fullscreen state of the window
+   * @param {boolean} isFullScreen - true if window is fullscreen
+   */
+  setFullscreenState: function (isFullScreen) {
```

Comment:
actually `setFrameFullScreen` is probably more consistent with other webview actions